.github/workflows/build.yml

```diff
@@ -104,51 +104,6 @@ jobs:
         run: |
           docker tag discourse/base:release${{ steps.arch-helper.outputs.arch_postfix_dash }} discourse/base:aarch64
           docker push discourse/base:aarch64
-  base_bookworm:
-    runs-on: ubuntu-20.04${{ ((github.event_name != 'schedule') && '-8core') || '' }}
-    timeout-minutes: ${{ github.event_name != 'schedule' && 30 }}
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          fetch-depth: 1
-      - name: build slim image
-        run: |
-          cd image && ruby auto_build.rb base_slim_bookworm
-      - name: tag slim images
-        id: tag-images
-        run: |
-          TAG=`date +%Y%m%d-%H%M`
-          echo "tag=$(echo $TAG)" >> $GITHUB_OUTPUT
-          docker tag discourse/base:build_slim_bookworm discourse/base:2.0.$TAG-slim-bookworm
-          docker tag discourse/base:build_slim_bookworm discourse/base:slim-bookworm
-      - name: build release image
-        run: |
-          cd image && ruby auto_build.rb base_bookworm
-      - name: tag release images
-        run: |
-          TAG=${{ steps.tag-images.outputs.tag }}
-          docker tag discourse/base:build_bookworm discourse/base:2.0.$TAG-bookworm
-          docker tag discourse/base:build_bookworm discourse/base:release-bookworm
-      - name: build test_build image
-        run: |
-          cd image && ruby auto_build.rb discourse_test_build_bookworm
-      - name: run specs
-        run: |
-          docker run --rm -e RUBY_ONLY=1 -e USE_TURBO=1 -e SKIP_PLUGINS=1 -e SKIP_LINT=1 discourse/discourse_test:build_bookworm
-      - name: Print summary
-        run: |
-          docker images discourse/base
-      - name: push to dockerhub
-        if: success() && (github.ref == 'refs/heads/main')
-        env:
-          DOCKERHUB_PASSWORD: ${{ secrets.DOCKERHUB_PASSWORD }}
-        run: |
-          TAG=${{ steps.tag-images.outputs.tag }}
-          docker login --username discoursebuild --password $DOCKERHUB_PASSWORD
-          docker push discourse/base:2.0.$TAG-slim-bookworm
-          docker push discourse/base:slim-bookworm
-          docker push discourse/base:2.0.$TAG-bookworm
-          docker push discourse/base:release-bookworm
```

Comment:
This is a temp job I introduced last week so that I can test the Debian Bookworm based image.